docs/subnets/when-to-use-subnet-vs-c-chain.md

```diff
@@ -11,6 +12,11 @@ with primary focus on EVM-based applications (for example, C-Chain,
 building an app on C-Chain versus [Subnet-EVM](https://github.com/ava-labs/subnet-evm), and help
 developers make more informed decisions.
 
+Subnets are subsets of the Avalanche Primary Network that validate a set of blockchains. Each Subnet's
```

Comment:
can we make it clear that the validator set of a subnet is a subset of the primary network validator set? This sentence is not clear as to what it means to be a subset ie. it could mean it only validates a subset of the blockchains.